mappings/net/minecraft/component/ComponentMapImpl.mapping

```diff
@@ -1,20 +1,20 @@
 CLASS net/minecraft/class_9335 net/minecraft/component/ComponentMapImpl
 	FIELD field_49654 baseComponents Lnet/minecraft/class_9323;
-	FIELD field_49655 extraComponents Lit/unimi/dsi/fastutil/objects/Reference2ObjectMap;
+	FIELD field_49655 changedComponents Lit/unimi/dsi/fastutil/objects/Reference2ObjectMap;
 	FIELD field_49656 copyOnWrite Z
 	METHOD <init> (Lnet/minecraft/class_9323;)V
 		ARG 1 baseComponents
 	METHOD <init> (Lnet/minecraft/class_9323;Lit/unimi/dsi/fastutil/objects/Reference2ObjectMap;Z)V
 		ARG 1 baseComponents
-		ARG 2 extraComponents
+		ARG 2 changedComponents
 		ARG 3 copyOnWrite
 	METHOD equals (Ljava/lang/Object;)Z
 		ARG 1 o
-	METHOD method_57933 copyFrom (Lnet/minecraft/class_9323;)V
+	METHOD method_57933 setAll (Lnet/minecraft/class_9323;)V
```

Comment:
ItemStack.copyComponentsFrom also calls this, that could possibly be adjusted?

Reply:
done